ccd-definition/ComplexTypes/CaseToPostLRspec.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "ID": "CaseToPostLRspec",
+    "ListElementCode": "accessCode",
+    "FieldType": "Text",
+    "ElementLabel": "Access Code",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CaseToPostLRspec",
+    "ListElementCode": "respondentCaseRole",
+    "FieldType": "Text",
+    "ElementLabel": "Case Role",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CaseToPostLRspec",
+    "ListElementCode": "expiryDate",
+    "FieldType": "Date",
+    "ElementLabel": "Expiry Date",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CaseToPostLRspec",
+    "ListElementCode": "citizenCaseRole",
+    "FieldType": "Text",
+    "ElementLabel": "Citizen Case Role",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CaseToPostLRspec",
+    "ListElementCode": "pinUsedFlg",
```

Comment:
No need to add Flg in the name.
pinUsed as a YesNo field is enough to figure out its a flag.

Reply:
Changed the name as suggested